hubstorage/client.py

```diff
@@ -162,6 +170,16 @@ def close(self, timeout=None):
         if self._batchuploader is not None:
             self.batchuploader.close(timeout)
 
+    @property
+    def USERAGENT(self):
+        warnings.warn(USERAGENT_DEPRECATED_MSG, DeprecationWarning)
```

Comment:
Looks like current tests were created around older testing environment which is outdated now and doesn't work as is even after a few fixes, I propose to recover tests in a separate PR.